doc/developers/contributing.rst

```diff
@@ -846,6 +846,10 @@ Finally, follow the formatting rules below to make it consistently good:
 * When editing reStructuredText (``.rst``) files, try to keep line length under
   80 characters when possible (exceptions include links and tables).
```

Comment:
Actually I see that we could request <= 88 characters nowadays

Reply:
I would rather not change this.

Reply:
OK fair enough. Make this point resolved.